src/main/java/io/strimzi/kafka/bridge/converter/MessageConverter.java

```diff
@@ -32,18 +32,20 @@
 	 *
 	 * @param kafkaTopic Kafka topic for sending message
 	 * @param message message to convert
+	 * @param partition partition of topic where the messages are sent when partition is specified in the URL
 	 * @return Kafka record
 	 */
-	KafkaProducerRecord<K, V> toKafkaRecord(String kafkaTopic, M message);
+	KafkaProducerRecord<K, V> toKafkaRecord(String kafkaTopic, M message, Integer partition);
```

Comment:
I would specify the `partition` parameter right after the `kafkaTopic`. They are related somehow.